.eslintrc.json

```diff
@@ -7,6 +7,7 @@
     "mocha": true
   },
   "rules": {
+    "linebreak-style": 0,
```

Comment:
Why did you add this new rule?

Reply:
to stop eslint linebreak errors on windows. It enforces the usage of Windows line endings: \r\n for CRLF.

Reply:
awesome